Ruby/FizzBuzz.rb

```diff
@@ -1 +1,13 @@
-100.times{|index| puts (index+1)%15==0?"FizzBuzz":(index+1)%3==0?"Fizz":(index+1)%5==0?"Buzz":index+1}
+#!/usr/bin/env ruby -w
```

Comment:
different file name

Reply:
I didn't modify them. They were already modified when I forked the project.